### ISSUE TITLE
Support: How can I change where Chezmoi stores its source state? I want to change the default of "~/.local/share/chezmoi"

### ISSUE THREAD

**Issue by fmotion1** [twpayne/chezmoi#3239]
## What exactly are you trying to do?

I've been reading the quick start tutorial here: https://www.chezmoi.io/quick-start/#concepts

> Assuming that you have already [installed chezmoi](https://www.chezmoi.io/install/), initialize chezmoi with:

`chezmoi init`

> This will create a new git local repository in: 

`~/.local/share/chezmoi` 

> where chezmoi will store its source state. By default, chezmoi only modifies files in the working copy.

What I am looking to do is change this initialization path from `~/.local/share/chezmoi` to something like `D:\Dev\.config`, or `D:\Dev\.chezmoi`. I really prefer to keep all of my development related content as organized and centralized as possible.

## What have you tried so far?

Asked some people on Discord, to which they responded that they had no idea if this was possible. So I started this thread.

## Where else have you checked for solutions?

* [x] I have read [chezmoi's user guide](https://chezmoi.io/user-guide/command-overview/), and not found the answer.
* [x] I have searched [chezmoi's reference guide](https://chezmoi.io/reference/), and not found the answer.
* [ ] Other, please give details.

## Additional context

TL;DR: How can I change the default repository path from `~/.local/share/chezmoi` to something of my choosing?

**Comment by halostatue** [comment]
You want to change `chezmoi.$FORMAT` configuration to set `sourceDir` to `D:\Dev\chezmoi`.

```yaml
sourceDir: 'D:\\Dev\\chezmoi'
```

You can test this with the [`-S`](https://www.chezmoi.io/reference/command-line-flags/global/#-s-source-directory) flag before changing it.
